src/posts/published/all-the-steps-needed-to-get-active-model-serializers-to-work-with-jsonapiadapter-and-jsonapiserializer-in-ember/index.md

```diff
@@ -7,7 +7,7 @@ subtitle: 'or the Headache That I Thought Would Never Go Away'
 title: 'All the Steps Needed To Get Active Model Serializers to Work with JSONAPIAdapter and JSONAPISerializer in Ember'
 ---
 
-Recently, I've been working every day on my project "Tempo", but it's been very slow with a lot of headaches. This always happens the first time or two you put theory to practice and you run into all the tiny hurdles and obstacles you weren't expecting.
+Recently, I've been working every day on my project "Tempo", but it's been very slow with a lot of headaches. This always happens the first time or two you put the theory to practice and you run into all the tiny hurdles and obstacles you weren't expecting.
```

Comment:
In my opinion, this isn't a typo, because there is no previously mentioned theory, there is no _the_ theory to which the article refers. In this case, I'm using the phrase to indicate taking any imaginary thing and making it real, not a specific imaginary thing. Please revert.

Reply:
How about "a theory" which sounds more pleasing?

Reply:
I think I'm just going to rewrite the sentence (and the next sentence). I personally don't mind it as is, but I understand why it might bother you. Please revert. I'll make the edits after we get this PR merged.